ex1.py

```diff
@@ -0,0 +1,18 @@
+
+try:
+    def calculate_average ():
+        counter = 0
+        sum = 0
+        function_stop = "q"
```

Comment:
add as const in beginnig of file.

FUNCTION_STOP_CHAR = "q"

Reply:
add the const in the beginning of the file or the beginning of the function?

Reply:
of the file